ethereum/ethpow.py

```diff
@@ -15,10 +16,9 @@
 try:
     import pyethash
     ETHASH_LIB = 'pyethash'  # the C++ based implementation
-    log.warn('using C++ implementation')
 except ImportError:
     ETHASH_LIB = 'ethash'
-    log.warn('using pure python implementation')
+    warnings.warn('using pure python implementation')
```

Comment:
Please use the `ImportWarning` category.
https://docs.python.org/2/library/warnings.html#warnings.warn
https://docs.python.org/2/library/warnings.html#warning-categories